### ISSUE TITLE
Not able to setup integration - error code 405

### ISSUE THREAD

**Issue by lukaskroczek** [JoDehli/PyLoxone#192]
**Which Model do you use from loxone? Miniserver? Gen1 or Gen2?** Gen2

**Which software version does your loxone use?** 14.2.6.16

**How did you install HomeAssistant? Over Hassio or manual install?** Hassio

**Which Version do you use of HomeAssistant?** 
Home Assistant 2023.8.1
Supervisor 2023.07.1
Operating System 10.4
Frontend 20230802.0 - latest

**Describe the bug**
When trying to configure integration, I receive error code 405 into log.

I tried url inputted into "miniserverIP" field. Also, I try credentials. I also tried http://192.168.10.150/jdev/cfg/apiKey endpoint (200 OK).

**Paste the error log with the following settings:**
```yaml

Logger: homeassistant.config_entries
Source: custom_components/loxone/api.py:84
Integration: PyLoxone (documentation, issues)
First occurred: 15:42:49 (2 occurrences)
Last logged: 16:13:40

Error setting up entry PyLoxone for loxone
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 388, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/__init__.py", line 195, in async_setup_entry
    if not await miniserver.async_setup():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/miniserver.py", line 140, in async_setup
    request_code = await self.lox_config.getJson()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/loxone/api.py", line 122, in getJson
    api_resp = await client.get("/jdev/cfg/apiKey")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1757, in get
    return await self.request(
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1530, in request
    return await self.send(request, auth=auth, follow_redirects=follow_redirects)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1617, in send
    response = await self._send_handling_auth(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1645, in _send_handling_auth
    response = await self._send_handling_redirects(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1703, in _send_handling_redirects
    raise exc
  File "/usr/local/lib/python3.11/site-packages/httpx/_client.py", line 1685, in _send_handling_redirects
    await hook(response)
  File "/config/custom_components/loxone/api.py", line 84, in raise_if_not_200
    raise LoxoneHTTPStatusError(
custom_components.loxone.api.LoxoneHTTPStatusError: Miniserver response code 405

```

**Comment by JoDehli** [comment]
@lukaskroczek have you used the same user and password when you tried it over the web browser? 

You can try a diferent user and password with pyloxone. 

In the doc there are some error codes described.

Code
○ 200 = everything okay
○ 403 = Miniserver is not registered with Loxone, CloudDNS disabled
○ 405 = Miniserver not reporting to CloudDNS
○ 409 = Unsecure Password in place, external access blocked.
○ 412 = port not opened
○ 418 = Denied, see responseJSON-Attribute for details
○ 481 = Miniserver could not connect to the remote connect service
○ 482 = Miniserver connection via remote connect was aborted due to a timeout
○ 483 = Miniserver makes a scheduled restart

**Comment by JoDehli** [comment]
@lukaskroczek please try the local IP address not a cloud service.

**Comment by lukaskroczek** [comment]
HI @JoDehli,

I am running on newest version (ok, it looks like a few hours ago they released a newer version). 

Yes, I tried same credentials. In fact, I tried 2 different credentials because for 1st time I recon there is some issue with my API account (this API user is correctly used by node-red-contrib-loxone library). So I tried another user and I receive same error code.

I was also looking for this error code in loxone and I do not understand what this "Miniserver not reporting to CloudDNS" means because clouddns is working.
